pkgs/applications/version-management/gitoxide/default.nix

```diff
@@ -4,29 +4,27 @@
 , cmake
 , pkg-config
 , stdenv
-, libiconv
-, Security
-, SystemConfiguration
```

Comment:
We're still using these for darwin in the `buildInputs` (at least until someone can properly verify that `gitoxide` works without), so we shouldn't delete them. OfBorg is failing on this.

Reply:
Oh.  My bad. Sorry about forgetting to check the CI again. Thank you for your kind notes here. :pray: